src/ServicePulse.Host/app/js/views/archive/controller.js

```diff
@@ -72,35 +73,53 @@
             vm.loadingData = false;
         };
 
+        var saveGroupOption = (amount, unit) => {
+            $cookies.put('archive_amount', amount);
+            $cookies.put('archive_unit', unit);
+        };
+
+        var getDefaultGroup = () => {
+            var amount = $cookies.get("archive_amount");
+            var unit = $cookies.get("archive_unit");
+
+            return {
+                amount: amount,
+                unit: unit
+            }
+        };
+
         var init = function () {
 
             vm.configuration = sharedDataService.getConfiguration();
             vm.error_retention_period = moment.duration(vm.configuration.data_retention.error_retention_period).asHours();
             vm.total = 1;
             vm.archives = [];
             vm.sort.page = 1;
-            vm.sort.start = moment.utc().subtract(vm.timeGroup.amount, vm.timeGroup.unit).format('YYYY-MM-DDTHH:mm:ss');
-            vm.sort.end = moment.utc().format('YYYY-MM-DDTHH:mm:ss');
 
+            var defaultGroup = getDefaultGroup();
+
+            vm.timeGroup.amount = defaultGroup.amount;
+            vm.timeGroup.unit = defaultGroup.unit;
+            vm.selectTimeGroup(vm.timeGroup.amount, vm.timeGroup.unit);
```

Comment:
The `selectTimeGroup` sets the `vm.timeGroup.amount` and `vm.timeGroup.unit` properties, so we can remove the 2 previous lines and make this a bit cleaner.